e2e-tests/controller-spark/controller_spark_sql_validation.sh

```diff
@@ -356,7 +356,9 @@ clear
 
 add_resource
 update_resource
-sleep 10
+# Provide enough buffer time before triggering the incremental run so that the previous full run
+# completes fully (including creation of hive tables)
+sleep 60
```

Comment:
Sometimes it is observed that the incremental run does not gets triggered because the previous full run is still in progress (this is because we only wait for generation of parquet files and not the creation of hive tables). One such example is this [build failure](https://ff3374279beb0c9b34708ea31d07a39db7f25084ae26d62a05a5c89-apidata.googleusercontent.com/download/storage/v1/b/cloud-build-gh-logs/o/log-b5056f60-0f4a-4a0a-a4d9-94ad1018d88f.txt?jk=ASOpP9gaDsevegR6H21KWVsBiPZw_NnweQRuvXmKms43rJdcV77acJO-G5Y_qoyGjSmDEfhKRpBcja6uzsSeypgYQFdL92zTr6NnOQf8py7ijlgQ1e8Vu-_QRADE0tmWHXx-8bbmebRsQANXHBtLjhYuAPue3AyjNZHUN0wdHsKG41r-Mq7YxvbyLDTQxdgsPNTvcic8AasH0M3Lc9JWDHR5OuVO707YUHL7PfejpllYDx8OjNL2PaZdTXfABZvxOUWturqhpsXHGBQ7hNLTQftSfZy6Yrxx4UyfJfI3OU68g7DYQm1u6UM_wz1wJPYx7M1xoctBpb3mocnPPRxl0MFQH9oCVtPc7z1KrLSvVyJ8sAAZ5MFKsw1cpyWP0yllq7ReTzpVkI9qckIM57Uqc3ZOzOWkBJPOW4op2wravNgxX21V_eInfq80e0-YCrv0M6iv4AlZr1Yc9UovXEzo9b1JNfQP3msyzR7HrFUBf1wJ48DMV-g90sVPXx70m1EfdR04oBsSH5f3hZ745Z_BlbfP-Hp7o8fZfjGd_TZVsv-HhI1-MDjjwwrudW12BvOtOsRTLlCB1ayED6BXQmUsVfrvbqa64Rw8CnDI38OEpgtJ87k8wJFOpWLtK5uJ4taztmjpMYasg80LpYtwnaoL8SKoc9bgUc8VKX0eFKbWFgw0NgTTTygK1Gc3VAuQfALz6ktml0h2pL3X_wH7BwjazJMFLrW5QZsLjOXtCv4LexrQJoxQGMcwlBfHUhQYwqFU3JDicWUbIe4lottjiQ3b40yGfhUNVeaWxD0uCxq2Qp00R_e8G2t7z_-BtJCFDRgi3kXqSAxCrR3y6jDJRprmUylc9kecnLUONravRUjJXjZWhkyju0k9k0049MdWR4QK86dtQaPfL9DKpGdfsY7rVqdBj9Jwwyah_mgfpysQmJ5PvevpbZwXttjYybhUShbfRfkHd5MarjneFKtCpVdG0iwVS7UaMj8bhScIt5NR2MByjnkZW18jlebvaHMarCi35fAqKScDGyebq1I8QRksfSi21_uTWZ42pDDkrhY8lyT0LY2yDz78-ZRhzHwa7D55frUT_0D9l4RgSk_h06lAWE8DX6d0b2h0s-TomRU9ZK1hq-Ys6mYJCeMSF5Kbgd2lH3bS-7jX1pjMTX9LaF1NPsQyIxjxiqO4nwn3o5iuCd7JA70FhKt9ouG_3lNmxEF2i0Vovke-9vITnnPicRKmkmMpzNs2zyHVBIe1Qj4&isca=1).